config/packages/overrides/platformsh.php

```diff
@@ -105,7 +105,7 @@
     }
 }
 
-if (isset($relationships['solr'])) {
+if (isset($relationships['solr']) && ($_SERVER['SEARCH_ENGINE'] ?? false) === 'solr') {
```

Comment:
if a defined parameter is a string, then setting `bool` as a fallback doesn't make any sense, even if it works
```suggestion
if (isset($relationships['solr']) && ($_SERVER['SEARCH_ENGINE'] ?? '') === 'solr') {
```